content/en/role-based-access-control.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Role-Based Access Control (RBAC)
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+
+The method of regulating access to computer or network resources based on individual users' roles within an organization.
```

Comment:
Hi @juliafmorgado 

I am not sure the current resource targets (`computer or network resources`) of access control fully cover targets that can be handled by RBAC. 

For instance, RBAC can be applied to various resources (objects) in Kubernetes such as Pods or Services.

Reply:
What about "regulating systems and resources access"?

Reply:
@juliafmorgado 
Yes, it looks fine for me.